external/source/rdll_template/README.md

```diff
@@ -0,0 +1,27 @@
+# Reflective DLL Project Template
+This is a project template for Microsoft's Visual Studio to aid in the creation
+of Reflective DLLs.
+
+## Installation
+To install the template, copy the template file to the Visual Studio templates
+directory. For a default Visual Studio 2019 installation the command would be:
+
+```
+copy "Reflective DLL.zip" "%USERPROFILE%\Documents\Visual Studio 2019\Templates\ProjectTemplates"
+```
+
+After the template has been copied, restart Visual Studio and select "Reflective
+DLL" from the available project templates.
```

Comment:
Should mention on Visual Studio 2019 and most Visual Studio versions you need to click `Create a New Project` first. Break this down into step by step instructions as some people may not be familiar with this.

Reply:
Also if we can add pictures here this would help a lot imho. Happy to assist with this if needs be.

Reply:
New directions look good, thanks for including them!